### DISCUSSION TITLE
Need help to whitelist a domain/website from adblock in Mobile Version

### DISCUSSION THREAD

**Discussion by GodsMasterpiece509** [bromite/bromite#2568] [General]
I looked at a few places but can't seem to find a way to whitelist a website to allow ads as i still want to have it blocked for other ones just this one Thanks In Advance

**Comment by Leopold702** [comment]
Ads can be enabled under website permissions.

**Reply by GodsMasterpiece509** [reply]
Hmm 🤔 would that slow me to whitelist the website ?

**Reply by GodsMasterpiece509** [reply]
> Ads can be enabled under website permissions.

![IMG_20230218_225135.jpg](https://user-images.githubusercontent.com/118909295/219922963-887c1310-5067-487a-bf25-30ffa1e8999d.jpg)

**Reply by GodsMasterpiece509** [reply]
> Ads can be enabled under website permissions.

Forgot to mentioned that i do have Adguard in Private DNS in settings too so even if i slow and change that setting how you mentioned it still doesn't allow me
